driver-core/src/main/java/com/datastax/driver/core/AggregateMetadata.java

```diff
@@ -122,7 +123,7 @@ static AggregateMetadata build(KeyspaceMetadata ksm, Row row, VersionNumber vers
     private static String makeStateFuncFullName(String stateFuncSimpleName, DataType stateType, List<DataType> argumentTypes) {
         List<DataType> args = Lists.newArrayList(stateType);
         args.addAll(argumentTypes);
-        return Metadata.fullFunctionName(stateFuncSimpleName, args);
+        return fullFunctionName(stateFuncSimpleName, args);
```

Comment:
In cases like that I find static imports make the code harder to read, especially outside of an IDE.